frontend/src/app/register/page.tsx

```diff
@@ -0,0 +1,88 @@
+"use client";
+
+import { useEffect } from "react";
+import styles from "./register.module.css";
+import { useSearchParams } from "next/navigation";
+import axios from "@/axios/client";
+import { ResponseCredential, ServerData } from "@/types/webAuthn";
+
+export default function RegisterUser() {
+  const searchParams = useSearchParams();
+
+  useEffect(() => {
+    async function f() {
+      // Decode token & check validity
+      const token = searchParams.get("token");
+
+      // TODO: verify token is valid in backend during registerRequest
+
+      const resp = await axios.get(`/api/auth/registerRequest`);
```

Comment:
I know this is what the example used, but could we change the names of these routes?
`/api/auth/registerRequest` -> `/api/auth/register`
`/api/auth/registerResponse` -> `/api/auth/register/verify` or `/api/auth/register/finish`
and then later, for authentication:
`/api/auth/authenticate`
`/api/auth/authenticate/verify` or `/api/auth/authenticate/finish`

Reply:
no plz

Reply:
Another small change, since the endpoint is customizable we should make the base url in [client.ts](https://github.com/lynx-locks/lynx-web/pull/15/files/277d3ac54a6a27a72fabdce2a0fd794c8ce7807a#diff-1c7b0bd5b820db0dc167ae2b5b90063641dafd1e5ff87b2be4d73c01f9944271) `http://localhost:5001/api/auth` and leave these endpoints as simply `/register`, `/register/verify`, etc.
This way, if someone wants to use a custom domain/routes for their webauthn, they aren't forced to use the full `/api/auth/` base. 

For example, they might want it to be something custom like `https://example.com/api/v1/lynx-auth/register` and `https://example.com/api/v1/lynx-auth/register/verify`.

Reply:
sg